### ISSUE TITLE
'vagrant up' failing after vmware-iso created and vagrant box added.

### ISSUE THREAD

**Issue by derhornspieler** [rapid7/metasploitable3#167]
<img width="738" alt="screen shot 2017-07-18 at 10 25 53 pm" src="https://user-images.githubusercontent.com/15236687/28347934-217a5372-6c08-11e7-92dd-7fa9bb914881.png">

**Comment by derhornspieler** [comment]
I've added whitelist feature per https://www.vagrantup.com/docs/vmware/boxes.html#ethernet-pcislotnumber instructions.  Rebuilding.  This time when it ran, Windows Server was trying to repair itself and this was the cause of the timeout.  Trying to rebuild with new Vagrantfile too see if this fixes the issue.

**Comment by Chan9390** [comment]
I dont have VMware setup on my system, so I couldnt test the changes you have mentioned. Does adding the above resolve the issue ?

**Comment by derhornspieler** [comment]
@Chan9390 didn't fix the issue unfortunately.  Hopefully this bug will be fixed soon.  After the iso is created, it keeps booting into system recovery.

<img width="813" alt="screen shot 2017-07-19 at 8 05 20 am" src="https://user-images.githubusercontent.com/15236687/28366056-0a3a27d6-6c59-11e7-9dd2-60ec2871b278.png">

**Comment by Chan9390** [comment]
Any updates ?

**Comment by jschulist** [comment]
I also encountered a BSOD STOP 0x7B after pulling down 8d6fbd4 and building the vmware-iso manually and running vagrant up.  Using Vagrant 2.0.1 & VMWare Fusion 10.0.1

After reviewing and comparing the logs from running a vagrant up vs running the VM image directly (which worked fine during my tests), I found that the linked clones created during a vagrant up are defaulting scsi0.virtualDev to "lsilogic" whereas the VM image built during packer build was configured (as per the vmx_data in the packer json file) with "lsisas1068".  This discrepancy appears to be the cause of the BSOD.  If I override scsi0.virtualDev via config.vm.provider.vmx in the Vagrantfile to "lsisas1068" then the VM boots up without issue.  It's an easy enough fix to override in the Vagrantfile but one that I wouldn't think should be necessary considering that the base VM and associated .vmx file correctly define scsi0.virtualDev as "lsisas1068".  Any idea why the linked clone is falling back to the older "lsilogic" setting?
